uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/HRContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -1,122 +1,127 @@
-          <div class="crm_contact_action_menu__bottom_margin">
-            <button class="btn btn-primary-outline">
-              <span>
-                <i class="fa fa-envelope-o"></i>
-              </span>
-              Send Welcome Email
-            </button>
-          </div>
-          <div class="crm_contact_action_menu__bottom_margin">
-            <button class="btn btn-primary-outline">
-              <span>
-                <i class="fa fa-envelope-o"></i>
-              </span>
-              Send Password Reset Email
-            </button>
-          </div>
-          <h3>User Has Access To:</h3>
-          <p>
-            <span class="crm_contact_action_menu__grey_text">No staff members</span>
-            <span class="fa fa-question-circle crm_contact_action_menu__help_icon"></span>
-          </p>
-          <button class="btn btn-secondary-outline">Manage Regional Access</button>
-        </div>
-        <div class="row crm_contact_action_menu__control_buttons_container">
-          <hr>
-          <div class="col-md-6">
-            <button class="button__delete btn-warning">Disable User Account</button>
-          </div>
-          <div class="col-md-6">
-            <button class="button__disable btn-danger">Delete User Account</button>
+<div id="crm-contact-actions-wrapper" data-edit-params='{ldelim}"cid": "{$contactId}", "class_name": "CRM_Contact_Page_Inline_Actions"{rdelim}'>
+  {crmButton id="crm-contact-actions-link" href="#" icon="bars"}
+    {ts}Actions{/ts}
+  {/crmButton}
+  <div id="crm-contact-actions-list" class="crm_contact-actions">
+    <div id="bootstrap-theme">
+      <div class="crm_contact-actions__body">
+        <div class="col-md-4 crm_contact-actions__body__user-info crm_contact-actions__user-info--margin">
+          <div class="col-md-12">
+            <h3>User Informations:</h3>
+            <p><span class="crm_contact-actions--bold">User:</span> <a href="#" class="text-primary"> 100 Lauren Ipsum </a></p>
+            <p><span class="crm_contact-actions--bold">Roles:</span> <span class="crm_contact-actions--dark-grey"> civihr_staff, civihr_manager </span></p>
+            <div class="crm_contact-actions--bottom-margin">
+              <button class="btn btn-primary-outline crm_contact-actions__btn">
+                <i class="fa fa-envelope-o"></i>Send Welcome Email
+              </button>
+            </div>
+            <div class="crm_contact-actions--bottom-margin">
+              <button class="btn btn-primary-outline crm_contact-actions__btn">
+                <i class="fa fa-envelope-o"></i>Send Password Reset Email
+              </button>
+            </div>
+            <h3>User Has Access To:</h3>
+            <p>
+              <span class="crm_contact-actions--dark-grey">No staff members</span>
+              <span class="fa fa-question-circle crm_contact-actions--primary"></span>
```

Comment:
what's this `crm_contact-actions--primary` class for? Please add screenshot

Reply:
This adds blue color for the help icon as here:
<img width="276" alt="screen shot 2018-01-24 at 2 23 08 pm" src="https://user-images.githubusercontent.com/6307362/35322396-3b860c58-0112-11e8-87d2-c33332ab33b4.png">

Reply:
1. Why isn't the icon blue by default?
2. If you really need to force the color, then as usual use the parent

Reply:
> Why isn't the icon blue by default?

Because it uses the default text color defined under namespace `#bootstrap-theme` $dark-gray (#4D4D69)`. So we need to force the blue color as in design.

Reply:
Then please follow my advice on point no. 2

Reply:
done!